### ISSUE TITLE
Circular routing

### ISSUE THREAD

**Issue by Sinnersslayer** [EasyFarm/EasyFarm#293]
It seems like circular routing doesnt work.
When bot reaches the last point on the route he turns back instead of begining from the first.

**Comment by mbout4it** [comment]
Looks like when you save the file route gets saved as a straight line regardless of the button.  Open the file and change it false.

,"StraightRoute":true,

**Comment by Mykezero** [comment]
Yes, this appears to be broken. There is no code in the program to actually update that flag, so the option is complete useless...
